### PR TITLE
Bump mysql strict encrypt connector

### PR DESCRIPTION
**Pull request by benmoriceau**
## What
Bump the connector version to make sure that the state lifecycle manager is included.

### CODE REVIEW

File: airbyte-integrations/connectors/destination-mysql-strict-encrypt/Dockerfile [hunk 1]
```diff
@@ -16,5 +16,5 @@ ENV APPLICATION destination-mysql-strict-encrypt
 
 COPY --from=build /airbyte /airbyte
 
-LABEL io.airbyte.version=0.1.4
```
Comment (benmoriceau) [inline airbyte-integrations/connectors/destination-mysql-strict-encrypt/Dockerfile:19]: the fact that it is not 0.1.5 is fishy

Comment (gosusnp) [reply airbyte-integrations/connectors/destination-mysql-strict-encrypt/Dockerfile:19]: Are we missing some commits somewhere?

Comment (lmossman) [reply airbyte-integrations/connectors/destination-mysql-strict-encrypt/Dockerfile:19]: It seems like the PR from the changelog that said it bumped it to 0.1.5 didn't actually bump this connector: https://github.com/airbytehq/airbyte/pull/12820/files

**Review by lmossman — APPROVED** [review]

**Comment by benmoriceau** [comment]
/publish connector=connectors/destination-mysql-strict-encrypt

**Review by gosusnp — APPROVED** [review]

**Comment by benmoriceau** [comment]
/publish connector=connectors/destination-mysql-strict-encrypt

**Comment by benmoriceau** [comment]
/publish connector=connectors/destination-postgres-strict-encrypt

**Pull request closed without merging**